app/src/androidTest/java/com/github/epfl/meili/profile/friends/UserInfoServiceTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.github.epfl.meili.profile.friends
+
+import com.github.epfl.meili.database.FirestoreDocumentService
+import com.github.epfl.meili.models.User
+import com.google.android.gms.tasks.OnSuccessListener
+import com.google.android.gms.tasks.Task
+import com.google.firebase.firestore.DocumentReference
+import com.google.firebase.firestore.DocumentSnapshot
+import com.google.firebase.firestore.FirebaseFirestore
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mockito.Mockito
+
+class UserInfoServiceTest {
```

Comment:
This should be in the unit tests since it doesn't use the UI, if I'm not mistaken

Reply:
Unfortunately, I already tried that and if I put this test in the unit test I am getting an error message saying that the initialisation of MockMaker failed. Do you have any idea how to fix that?

Reply:
You can add this in the app gradle to solve that.

`    testImplementation 'org.mockito:mockito-core:3.8.0'
`

Reply:
This line was already in the gradle, and I quickly tried a couple more things but without any success.